hypergraph/concepts/02-testnet.md

```diff
@@ -6,12 +6,7 @@ hide_table_of_contents: false
 ---
 <intro-end />
 
-Constellation provides developers with a full featured testnet to test applications and metagraphs before they're ready for the production environment. 
-The testnet has all of the same features as mainnet and can therefore be used to validate application integrations in a realistic way. 
-
-:::info Developing a metagraph
-Metagraph developers may use testnet as they're nearing production readiness. For initial development, developing on your own test cluster is recommended. See [Deploying to Testnet](/sdk/guides/deploy-testnet).
-:::
+Constellation provides developers with the testnet, a  testing environment where new protocol updates are released and tested before they reach production. Unlike the more stable integration net, the testnet is a  platform for developers wanting to experiment with the latest features and validate their applications' compatibility with upcoming changes. 
```

Comment:
Some awkward language here - could use a rewrite.

Reply:
Metagraph developers can use testnet as an experimental environment for testing metagraphs against the newest protocol updates before they are rolled out to integrationnet. Testnet provides a platform for developers to experiment with the latest features, enabling them to validate how their metagraphs will interact with upcoming changes.